monai/transforms/intensity/array.py

```diff
@@ -2325,5 +2327,5 @@ def __call__(self, mask: NdarrayOrTensor):
             h_map[h_map == region.label] = h_dist
             v_map[v_map == region.label] = v_dist
 
-        hv_maps = convert_to_tensor(np.stack([h_map, v_map]), track_meta=get_track_meta())
+        hv_maps = convert_to_tensor(np.concatenate([h_map, v_map]), track_meta=get_track_meta())
```

Comment:
@YuanTingHsieh could you please let me know what is the reason for this change? Thanks

Reply:
Since `h_map` and `v_map` already have channel dim, and stack will add one more dim.

Reply:
I see, so you are removing the dim from instance mask after calculating h_mav, and v_map. 👍